perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -1108,6 +1136,21 @@ public RichAttributeRowMapper(AttributeRowMapper attributeRowMapper, RowMapper<P
 		}
 	}
 
+        public List<Attribute> getAttributes(PerunSession sess, UserExtSource ues) throws InternalErrorException {
+		try {
+			return jdbc.query("select " + getAttributeMappingSelectQuery("ues") + " from attr_names " +
+					"left join user_ext_source_attr_values ues on user_id=? and ext_source_id=? " +
```

Comment:
Hi, please change all sql references to ``user_id`` and ``ext_source_id`` columns to single ``ues_id`` as mentioned in #1306.